README.md

```diff
@@ -144,6 +144,10 @@ different extension libraries. If this is necessary, you need to modify one of
 the libraries to call `register_type` on that pair of `BodyNode` and `Joint`
 types.
 
+## License
```

Comment:
Hmm, I think it's a bit weird for the `License` and `Authors` block to not be the same level heading (one is Heading 2 and the other is Heading 3).  Perhaps we should change one?

Reply:
I changed the level of `Authors` in #50. I just didn't want to cause a conflict. :smile:

Reply:
Ah ok, that makes sense.